spec/expression.dd

```diff
@@ -1100,6 +1100,60 @@ $(GNAME CastExpression):
         (foo) - p;      // subtract p from foo
         -------------
 
+$(H4 $(LNAME2 cast_basic_data_types, Basic Data Types))
+    $(P There are some cases where the $(DDSUBLINK spec/type, implicit-conversions, implicit conversion)
+        cannot be applied, so cast may be used to force the type system to accept the reinterpretation.
```

Comment:
Rephrase to: "For situations where $(DDSUBLINK spec/type, implicit-conversions, implicit conversions)" on basic types cannot be performed, the type system may be forced to accept the reinterpretation of a memory region by using a cast.